test/command_callback/test_javascript_tsserver_command_callback.vader

```diff
@@ -6,3 +6,11 @@ After:
 
 Execute(The default command should be correct):
   AssertLinter 'tsserver', ale#Escape('tsserver')
+
+Execute(should resolve correct path when nested 1):
+  call ale#test#SetFilename('tsserver_paths/src/level-1/level-2/file3.ts')
+  AssertLSPProject ale#path#Simplify(g:dir . '/tsserver_paths/src/level-1/tsconfig.json')
```

Comment:
```suggestion
  AssertLSPProject ale#path#Simplify(g:dir . '/tsserver_paths/src/level-1')
```

The paths are slightly off.

Reply:
I knew it! For some reason it kept failing on my machine when I left off the `tsconfig.json`. Strange.. But anyway, yeah changes look good.